ngraph/python/tests/test_ngraph/test_adaptive_pool.py

```diff
@@ -1,13 +1,11 @@
 import ngraph as ng
 import numpy as np
-from tests import xfail_issue_59935
 from tests.runtime import get_runtime
 
 
-@xfail_issue_59935
 def test_adaptive_avg_pool():
     runtime = get_runtime()
-    input = np.reshape([0, 4, 1, 3, -2, -5, -2,
+    input = np.reshape([0.0, 4, 1, 3, -2, -5, -2,
```

Comment:
nitpick: Do we really need this change?

Reply:
Yes, we do

Reply:
Another way, input will be interpreted as integer, and calculations will be wrong